flight/targets/naze32/fw/FreeRTOSConfig.h

```diff
@@ -81,7 +81,7 @@ NVIC value of 255. */
 
 /* Enable run time stats collection */
 #if defined(DIAG_TASKS)
-#define configCHECK_FOR_STACK_OVERFLOW	2
+#define configCHECK_FOR_STACK_OVERFLOW	0
```

Comment:
Unfortunately this pretty much breaks it.

Mode 1, the "cheap" check (inspecting the stack pointer), is broken with this.  Mode 2, the "expensive" check (checking the end of the stack for evidence of overflow), actually still works fine.  Unfortunately, using it implies also doing mode 1 in freertos.